dotnet/src/Planners/Planners.OpenAI/Stepwise/FunctionCallingStepwisePlanner.cs

```diff
@@ -21,52 +21,44 @@ public sealed class FunctionCallingStepwisePlanner
     /// <summary>
     /// Initialize a new instance of the <see cref="FunctionCallingStepwisePlanner"/> class.
     /// </summary>
-    /// <param name="kernel">The <see cref="Kernel"/> containing services, plugins, and other state for use throughout the operation.</param>
     /// <param name="config">The planner configuration.</param>
     public FunctionCallingStepwisePlanner(
-        Kernel kernel,
         FunctionCallingStepwisePlannerConfig? config = null)
     {
-        Verify.NotNull(kernel);
-        this._kernel = kernel;
-        this._chatCompletion = kernel.GetService<IChatCompletion>();
-
-        ILoggerFactory loggerFactory = kernel.LoggerFactory;
-
-        // Initialize prompt renderer
-        this._promptTemplateFactory = new KernelPromptTemplateFactory(loggerFactory);
-
-        // Set up Config with default values and excluded plugins
         this.Config = config ?? new();
-        this.Config.ExcludedPlugins.Add(RestrictedPluginName);
-
-        this._initialPlanPrompt = this.Config.GetPromptTemplate?.Invoke() ?? EmbeddedResource.Read("Stepwise.InitialPlanPrompt.txt");
+        this._generatePlanYaml = this.Config.GetPromptTemplate?.Invoke() ?? EmbeddedResource.Read("Stepwise.GeneratePlan.yaml");
         this._stepPrompt = this.Config.GetStepPromptTemplate?.Invoke() ?? EmbeddedResource.Read("Stepwise.StepPrompt.txt");
-
-        // Create context and logger
-        this._logger = loggerFactory.CreateLogger(this.GetType());
+        this.Config.ExcludedPlugins.Add(RestrictedPluginName);
```

Comment:
Can we change this to something more descriptive to the plugin, e.g., `StepwisePlannerPluginName`